scanpipe/tests/test_pipelines.py

```diff
@@ -860,6 +860,60 @@ def test_scanpipe_scan_codebase_pipeline_integration(self):
         expected_file = self.data / "scancode" / "is-npm-1.0.0_scan_codebase.json"
         self.assertPipelineResultEqual(expected_file, result_file)
 
+    def test_scanpipe_scan_codebase_creates_top_level_paths(self):
+        pipeline_name = "scan_codebase"
+        project1 = make_project()
+
+        filename = "is-npm-1.0.0.tgz"
+        input_location = self.data / "scancode" / filename
+        project1.copy_input_from(input_location)
+
+        run = project1.add_pipeline(pipeline_name)
+        pipeline = run.make_pipeline_instance()
+
+        exitcode, out = pipeline.execute()
+        self.assertEqual(0, exitcode, msg=out)
+
+        expected_top_level_paths = ["is-npm-1.0.0.tgz", "is-npm-1.0.0.tgz-extract"]
+
+        top_level_resources = project1.codebaseresources.filter(parent_path=None)
+        top_level_paths = [res.path for res in top_level_resources]
```

Comment:
When possible, use full variable name for better readability and consistency with the codebase.